AdMobAdapter/src/main/java/com/chartboost/mediation/admobadapter/AdMobAdapter.kt

```diff
@@ -205,16 +214,19 @@ class AdMobAdapter : PartnerAdapter {
             else COPPA_NOT_SUBJECT
         )
 
-        MobileAds.setRequestConfiguration(
-            MobileAds.getRequestConfiguration().toBuilder()
-                .setTagForChildDirectedTreatment(
-                    if (isSubjectToCoppa) {
-                        RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE
-                    } else {
-                        RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE
-                    }
-                ).build()
-        )
+        // There have been known ANRs when calling setRequestConfiguration() on the main thread.
+        CoroutineScope(IO).launch {
+            MobileAds.setRequestConfiguration(
+                MobileAds.getRequestConfiguration().toBuilder()
+                    .setTagForChildDirectedTreatment(
+                        if (isSubjectToCoppa) {
+                            RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE
+                        } else {
+                            RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE
+                        }
+                    ).build()
+            )
+        }
```

Comment:
nit: Unnesting some of this might make it a little more readable

```
val tag = if (isSubjectToCoppa) {
        RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_TRUE
    } else {
        RequestConfiguration.TAG_FOR_CHILD_DIRECTED_TREATMENT_FALSE
    }

MobileAds.getRequestConfiguration().apply {
    setTagForChildDirectedTreatment(tag)
}.toBuilder().let{  config ->
    MobileAds.setRequestConfiguration(config)
}
```

Reply:
OK I can do this for the upcoming release.